frontend/cache.lua

```diff
@@ -14,10 +14,16 @@ end
 
 local Cache = {
     -- Cache configuration:
-    -- Max storage space, in bytes
+    -- Max storage space, in bytes (or number of slots if unlimited_storage is true)
     size = 8 * 1024 * 1024,
-    -- Average item size is used to compute the amount of slots in the LRU
+    -- Average item size is used to compute the amount of slots in the LRU.
     avg_itemsize = 8196,
+    -- Should LRU not impose a storage space limitation, and only a slot count (in which case, size is taken to mean slots)?
+    -- c.f., CatalogCache
+    unlimited_storage = false,
```

Comment:
That's the kind of code (having `size` meaning something or some other thing depending on some other attribute, `unlimited_storage`) that I have often done while developping although it looks clumsy - but it eventually looked so clumsy/ugly I had to find another solution before proposing a PR :)

Reply:
The other solution is also an instance settings (i.e., slots instead of size).

Reply:
Indeed, ideally, there could be both (both optionnal?) `maxitems` and `maxsize` (or `slots` and `size`).

Reply:
Yeah, that actually makes the rest easier since `self.slots` actually already exists down the line ;).